### DISCUSSION TITLE
Secrets for CronJobs not showing on Orchestration Generator Payload

### DISCUSSION THREAD

**Discussion by rmuller-vu** [mia-platform/community#491] [General]
### Description

We have an issue with secrets not showing on the Mia Orchestrator Generator payload like we receive with microservices. Fig. 1 shows us adding the secret to the cronjob in the console. Fig. 2 shows the payload missing the secrets section. Fig. 3 shows secrets appearing in the payload for microservices.

Fig. 1 
![image](https://github.com/mia-platform/community/assets/136749338/ca2c48d6-ebbe-4562-8a54-538664b28fd2)

Fig. 2
![image](https://github.com/mia-platform/community/assets/136749338/bafd8276-5ff6-4f4a-818a-8798e4b2e88d)

Fig. 3
![image](https://github.com/mia-platform/community/assets/136749338/d3b74b81-f567-4e84-9201-90a23dc8b2ed)

**Comment by MIA-Deltat1995** [comment]
Hi @rmuller-vu, thank you for the issue. The problem is already known but not yet resolved, because configmaps and secrets are additional info for a cronjob config and don't affect the deployed manifest that is manually managed through the Console.

Please contact your Mia-Platform referent to track the bug so that we can take charge it.
